app/assets/javascripts/oregon_digital/toggle_login_footer.js

```diff
@@ -0,0 +1,10 @@
+function toggleFooter (footer) {
```

Comment:
All this could possibly be simplified to
```
$('#login-modal-footer-OSU-footer').hide();
$('#login-modal-footer-UO-footer').hide();
$('#login-modal-footer-local-footer').hide();
$('#login-modal-footer-' + footer).toggle();
```
Unless the `toggle()` transition isn't preserved and looks nice